Makefile

```diff
@@ -24,7 +24,8 @@ debug:
 
 .PHONY: lint
 lint:
-	exit
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.43.0
+	golangci-lint run ./...
```

Comment:
Didn't work locally (needed to brew install instead) but as CI is working this seems ok